docs/project-file-structure.md

```diff
@@ -64,30 +70,136 @@ to organize their structure
 
 Resources, scenes, scripts etc. used exclusively within that scene should be stored in the scenes subdirectory
 
+A scene and its script should always be together within the directory i.e. no /scripts/ and /scenes/ directories should exist within
+the project somewhere.
+
 Example:
 ```
 root/
 |- src/
-    |- Theme/
+    |- theme/
     |   |- ButtonNormal.tres
     |   |- ButtonHover.tres
     |   |- ButtonDisabled.tres
-    |- Scripts/
+    |- utility/
     |   |- Math.gd
-    |- Menu/
+    |- menu/
     |   |- MenuManager.tscn
     |   |- MenuManager.gd
     |   |- MainMenu/
     |   |   |- MainMenu.tscn
     |   |   |- MainMenu.gd
     |   |   |- MainMenuFont.ttf
     |   |   |- MainMenuLabel.tres
-    |   |- SettingsMenu/
+    |   |- settings_menu/
     |   |   |- Settings.tscn
     |   |   |- Settings.gd
     |   |   |- Resolution/
     |   |   |   |- ResolutionSelector.gd
-    |   |   |- Volume/
+    |   |   |- volume/
     |   |   |   |- VolumeGrid.tscn
     |   |   |   |- VolumeGrid.gd
 ```
+
+#### Naming Convention
+
+Directory names should be snake_case.
+```
+root/src/game_session/
+```
+
+File names should be PascalCase.
+```
+root/src/game_session/Map.tscn
+```
+
+### Asset Organization
+
+Asset files should be organized within a relevant category from root/
+```
+root/art/
+root/music/
+root/sfx/
+```
+
+Asset files should be organized in a subdirectory that specifies it's relevancy within the application.
+```
+root/art/economy/
+root/art/military/
+root/sfx/economy/
+root/sfx/technology/
+```
+
+Asset files should be prefixed by the asset type
+```
+iconCoal.png
+modelSoldier.obj
+ostTheCrown.mp3
+videoIntro.mp4
+```
+
+The prefix can be removed if the file is within a subdirectory that replaces the asset type 
+```
+./iconCoal.png -> ./icon/Coal.png
+./ostTheCrown.png -> ./ost/TheCrown.mp3
+```
+
+#### Naming Convention
+
+Directories should be snake_case.
+
+Asset files should be PascalCase for their name.
+```
+TheCrown.mp3
+Coal.png
+```
+
+Asset files with a prefix should have that prefix be lowercase.
+```
+ostTheCrown.mp3
+iconCoal.png
+```
+
+### Data File Organization
+
+All data files should be in the root/common/ directory.
```

Comment:
My assumption is that it would not be apart of the executable. These files are separate but within the install directory. So it might look like this

```
root/OpenVic2.exe
root/common/...
```

If a modder has to download godot to make a mod, then I'd argue we have failed at making modding more accessible. Nothing is more accessible then opening and editing a text file directly, and obfuscating that behind the need to download an engine or other tool seems like a bad idea to me

Reply:
In Godot the application data and assets is stored in the pck file, this is the `res://` file path protocol, this then suggests we need a separate designation for external application files.

Reply:
I would suggest having a directory in our root that's excluded from project export, something like `data/` which refers to data files that can be modified outside the program, and it would reside in `OpenVic2/game/data`

Reply:
Yeah thats what common is already. Common is just whats already in the reqs so it'll be easier to just stick with that

Reply:
I'll just put it in here that it gets removed from the export